kodev

```diff
@@ -94,6 +94,10 @@ ${SUPPORTED_TARGETS}"
             assert_ret_zero $?
             ;;
         android)
+            if [ "$NDK" == "" ]; then
+                # Some distributions use `ANDROID_NDK` instead, fall back to it
```

Comment:
Yes, it should.

* `-z ${NDK+x}` (unset or empty)
* `-n ${ANDROID_NDK+x}` (set and non-empty)